fs/fstestutil/record/record.go

```diff
@@ -1,11 +1,11 @@
-package record // import "bazil.org/fuse/fs/fstestutil/record"
+package record // import "github.com/suffuse/fuse/fs/fstestutil/record"
```

Comment:
Ahh I see what you mean now. Then you should have probably left out manual the changes to `.travis.yml` and `README.md`.

Anyway. No need to argue about it. I understand the 'keep automated transformations in their own commit' philosophy'